example-theme/theme.edn

```diff
@@ -1,4 +1,4 @@
-{:font-family "sans-serif"
+{:font-family "'Lato', sans-serif"
```

Comment:
Should this be mentioned in the changelog?

Reply:
Like "new example font is Lato"?

Reply:
Yeah, "The example-theme now uses the Lato font"

Reply:
this has been addressed